### DISCUSSION TITLE
What is the problem? There was a warning error

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#1153] [Q&A] [answered]
I update for this Commits 

 
![未命名-2](https://user-images.githubusercontent.com/17884664/232381017-bce3f25f-4ed1-4139-9486-ae64a91e2a23.png)

now Warning prompt appears like this img

![未命名-5](https://user-images.githubusercontent.com/17884664/232376248-773878a4-a100-4431-9710-d3b4e341a280.png)

I suspect it was caused by the above modifications, but I am not entirely certain

I don't know where the problem is, it seems like there hasn't been such a warning before

**Comment by Danielku15** [comment, marked as answer]
There is no problem 😉 . These logs are some temporary additions I added to test and analyze certain things. I will likely remove them soon again.
